skin/base.css

```diff
@@ -103,8 +103,9 @@
 }*/
 
 /* hide private browsing indicator in the vertical tab box */
-#verticaltabs-box .private-browsing-indicator {
-  display: none;
+#verticaltabs-box .private-browsing-indicator,
+#verticaltabs-box #private-browsing-indicator {
```

Comment:
Wait, we have a private browsing indicator id as well as class?  Ugh, disappointing…